u_common.go

```diff
@@ -138,6 +140,8 @@ const (
 
 	// versions
 	helloAutoVers = "0"
+
+	DefaultWeights_Version = "0"
```

Comment:
Is there a specific reason we need it and want it to be exported? 

Also just some styling suggestions, we really don't want a name with underscore(`_`). Possibly name it `DefaultWeightVers` or something.

Reply:
We won't need it anymore.